Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,36 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Object Removal Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
+	{
+		if (m_LocalCoreStartupParameter.update)
+		{
+			// Set lock so codes can be enabled/disabled in game without crashes.
+			m_LocalCoreStartupParameter.done = false;
+			for (int current_object_removal_code = 0; current_object_removal_code < m_LocalCoreStartupParameter.num_object_removal_codes; current_object_removal_code++)
+			{
+				for (int d = 0; d < m_LocalCoreStartupParameter.num_object_skip_data_bytes[current_object_removal_code]; d++)
```

Comment:
What shall happen if "num_object_skip_data_bytes[..] != size"? Maybe just another "continue;"?

So the next code is the same as just a common memcmp().